include/openthread.h

```diff
@@ -191,6 +191,19 @@ otInstance *otInstanceInit(void);
 #endif
 
 /**
+ * This function initializes OpenThread stack.
+ *
+ * This function initializes OpenThread stack with following steps:
+ *   1. Initialize settings subsystem;
+ *   2. Retrieve active operational dataset from non-volatile memory, and apply the network configuration;
+ *   3. Retrieve pending operational dataset from non-volatile memory.
+ *
+ * @param[in] aInstance A pointer to an OpenThread instance.
+ *
+ */
+void otInit(otInstance *aInstance);
```

Comment:
The new `otInit()` is aim to void duplicate codes in the two different `otInstanceInit()` APIs.
I could move `otInit()` to be a local function, and be called by the two `otInstanceInit()`, do you think this way is better?